hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerScrubberConfiguration.java

```diff
@@ -17,55 +17,100 @@
+      "hdds.container.scrub.data.scan.interval";
+  static final String VOLUME_BYTES_PER_SECOND_KEY =
+      "hdds.container.scrub.volume.bytes.per.second";
+
+  static final long METADATA_SCAN_INTERVAL_DEFAULT =
+      Duration.ofHours(3).toMillis();
+  static final long DATA_SCAN_INTERVAL_DEFAULT =
+      Duration.ofDays(1).toMillis();
+  static final long BANDWIDTH_PER_VOLUME_DEFAULT =
+      (long) StorageUnit.MB.toBytes(1);
 
   @Config(key = "enabled",
       type = ConfigType.BOOLEAN,
       defaultValue = "false",
       tags = {ConfigTag.STORAGE},
       description = "Config parameter to enable container scrubber.")
-  private boolean enabled;
+  private boolean enabled = false;
 
   @Config(key = "metadata.scan.interval",
       type = ConfigType.TIME,
       defaultValue = "3h",
       tags = {ConfigTag.STORAGE},
-      description = "Config parameter define time interval in milliseconds" +
-          " between two metadata scans by container scrubber.")
-  private long metadataScanInterval;
+      description = "Config parameter define time interval" +
+          " between two metadata scans by container scrubber." +
+          " Unit could be defined with postfix (ns,ms,s,m,h,d).")
+  private long metadataScanInterval = METADATA_SCAN_INTERVAL_DEFAULT;
 
   @Config(key = "data.scan.interval",
       type = ConfigType.TIME,
-      defaultValue = "1m",
+      defaultValue = "1d",
       tags = {ConfigTag.STORAGE},
       description = "Minimum time interval between two iterations of container"
           + " data scanning.  If an iteration takes less time than this, the"
-          + " scanner will wait before starting the next iteration."
-  )
-  private long dataScanInterval;
+          + " scanner will wait before starting the next iteration." +
+          " Unit could be defined with postfix (ns,ms,s,m,h,d).")
+  private long dataScanInterval = DATA_SCAN_INTERVAL_DEFAULT;
 
   @Config(key = "volume.bytes.per.second",
-      type = ConfigType.LONG,
-      defaultValue = "1048576",
+      type = ConfigType.SIZE,
+      defaultValue = "1MB",
       tags = {ConfigTag.STORAGE},
       description = "Config parameter to throttle I/O bandwidth used"
           + " by scrubber per volume.")
-
-  private long bandwidthPerVolume;
-
+  private long bandwidthPerVolume = BANDWIDTH_PER_VOLUME_DEFAULT;
+
+  @PostConstruct
+  public void validate() {
+    if (metadataScanInterval < 0) {
+      LOG.warn(METADATA_SCAN_INTERVAL_KEY +
+              " must be >= 0 and was set to {}. Defaulting to {}",
+          metadataScanInterval, METADATA_SCAN_INTERVAL_DEFAULT);
+      metadataScanInterval = METADATA_SCAN_INTERVAL_DEFAULT;
+    }
+
+    if (dataScanInterval < 0) {
+      LOG.warn(DATA_SCAN_INTERVAL_KEY +
+              " must be >= 0 and was set to {}. Defaulting to {}",
+          dataScanInterval, DATA_SCAN_INTERVAL_DEFAULT);
+      dataScanInterval = DATA_SCAN_INTERVAL_DEFAULT;
+    }
+
+    if (bandwidthPerVolume < 0) {
```

Comment:
What is the effect of zero bandwidth?  Does it mean no limit?

Reply:
Here, I think the desired behaviour is make the scrubber disabled.
Currently, the data scanner will run, but it will just keep throttling on each scan since the limit is always hit. BTW, HDFS has a similar config and will disable the scrubber if 0 is given.
I think I'll let the scrubber be disabled if this a 0 is given, but 0 should be valid although.

Reply:
BTW, if we want to put no limit on this item, maybe we could define an extremely large value for it.

Reply:
This bandwidth only affects data scanner, so I'll disable the data scanners on a 0 bandwidth while keep metadata scanners running.